Client/src/Features/Auth/authSlice.js

```diff
@@ -14,7 +14,9 @@ export const register = createAsyncThunk(
   "auth/register",
   async (form, thunkApi) => {
     try {
-      const res = await axiosInstance.post("/auth/register", form);
+      // Convert email to lowercase before sending the request
+      const modifiedForm = { ...form, email: form.email.toLowerCase() };
```

Comment:
Can we move the `toLowerCase()` function to the register page?  Ideally we should pefrorm all operations we need to before handing things off to redux, redux should only be kept clean for state management only